library/src/armory/export/drise/__init__.py

```diff
@@ -0,0 +1 @@
+from armory.export.drise.exporter import DRiseSaliencyObjectDetectionExporter
```

Comment:
It allows
```python
from armory.export.drise import DRiseSaliencyObjectDetectionExporter
```
instead of having to import from `armory.export.drise.exporter` (because `exporter.py` is an internal implementation detail to separate/split code from `impl.py`)